bad.py

```diff
@@ -0,0 +1,4 @@
+5 == 5
+
+concat_list = ["str1" "str2", "str3"]
+6 == 6
```

Comment:
# <img src="https://r2c.dev/semgrep_icon.png" height=20px alt="" /> semgrep
**Severity:** Error :red_circle:
**Rule id:** [python.lang.correctness.useless-eqeq.useless-eqeq](https://semgrep.dev/r/python.lang.correctness.useless-eqeq.useless-eqeq)

This is always True: `6 == 6` or `6 != 6`. If testing for floating point NaN, use `math.isnan(6)`,
or `cmath.isnan(6)` if the number is complex.

<sub>[Report an issue with this rule.](https://github.com/returntocorp/semgrep-rules/issues/new?assignees=&labels=&template=rule_bug_report.md&title=python.lang.correctness.useless-eqeq.useless-eqeq)</sub>

Reply:
this is the message ([rule.id](https://semgrep.dev))


<div>
<a href="https://semgrep.dev">
<img align="right" src="r2c.dev/semgrep_icon.png" height=20px alt="" />
</a>


<sub> [How do I suppress this?](https://semgrep.dev/docs/running-rules/#managing-findings) **·** "
[Report a bug](https://semgrep.dev/docs/) **·** [Manage rule](https://semgrep.dev/manage/projects)</sub>
</div>